WPMedia/fusion-news-theme-blocks
# Update Feature generators for enhanced styling architecture

## Description

Update the generators to create block features as per 2.00 architecture

* Include `.scss` file
* Add className default boilerplate
* Updates tests to remove the need to mock everything
* Install `arc-themes-components` as a dev dependency 
* Update babel to React library in at runtime

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout update-generators-for-enhanced-styling`
2. `npm i`
3. Run `npm run generator:feature`
4. Enter `my-new-block`
5. Check the generator code is as expected, tests run `npm run test:watch`
6. Run `npm run generator:feature:feature`
7. Enter `my-new-block`
8. Enter `second-feature`
9. Check the generator code is as expected, tests run `npm run test:watch`


## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
